store/src/java/com/zimbra/cs/util/ProxyConfGen.java

```diff
@@ -1922,6 +1977,7 @@ public void update() {
     private static Map<String, String> mVars = new HashMap<String, String>();
     private static Map<String, ProxyConfVar> mDomainConfVars = new HashMap<String, ProxyConfVar>();
     static List<DomainAttrItem> mDomainReverseProxyAttrs;
+    static Set<String> mListenAddresses = new HashSet<>();
```

Comment:
Shouldn't the RHS be "new HashSet<String>()"? or "Sets.newHashSet()"?

Reply:
This form compiles fine so I suppose it's up to whatever style we wish to follow.
The compiler is filling in the appropriate type to the right hand side.

Reply:
Perhaps also need to specify the type in new HashSet<String>(); like it is specified in the preceding lines.

Reply:
Done.